src/Estimators/include/BipedalLocomotion/Estimators/FloatingBaseEstimatorParams.h

```diff
@@ -0,0 +1,182 @@
+/**
+ * @file FloatingBaseEstimatorParams.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_PARAMS_H
+#define BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_PARAMS_H
+
+#include <Eigen/Dense>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+namespace FloatingBaseEstimators
+{
+/**
+* @brief Struct containing sensor measurement deviation parameters of floating base estimators
+*
+*/
+class SensorsStdDev
```

Comment:
use `struct` for public access.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069